src/components/ImportJobs/components/FileUpload/FileUpload.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import noop from 'lodash/noop';
+
+import { Button } from '@folio/stripes/components';
+
+import css from './FileUpload.css';
+
+class FileUpload extends React.Component {
+  static propTypes = {
+    title: PropTypes.string.isRequired,
+    uploadBtnText: PropTypes.string.isRequired,
+  };
+
+  render() {
+    const {
+      title,
+      uploadBtnText,
+    } = this.props;
+
+    return (
+      <div className={css.upload}>
+        <span className={css.uploadTitle}>{title}</span>
+        <Button
+          buttonStyle="primary"
+          onClick={noop}
```

Comment:
Do we really need to include _ noop? can we simple use javascript void?

Reply:
I think it is good to be consistent with such things. As this way is already used in another part of the application.